Authenticator/Source/TokenListViewController.swift

```diff
@@ -44,44 +44,48 @@ class TokenListViewController: UITableViewController {
 
     private var displayLink: CADisplayLink?
     private let ring: OTPProgressRing = OTPProgressRing(frame: CGRectMake(0, 0, 22, 22))
-    private let noTokensLabel = UILabel()
+    private lazy var noTokensLabel: UILabel = {
+        let noTokenString = NSMutableAttributedString(string: "No Tokens\n",
+            attributes: [NSFontAttributeName: UIFont(name: "HelveticaNeue-Light", size: 20)!])
+        noTokenString.appendAttributedString(NSAttributedString(string: "Tap + to add a new token",
+            attributes: [NSFontAttributeName: UIFont(name: "HelveticaNeue-Light", size: 17)!]))
+        noTokenString.addAttributes([NSFontAttributeName: UIFont(name: "HelveticaNeue-Light", size: 25)!],
+            range: (noTokenString.string as NSString).rangeOfString("+"))
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 107 characters (line_length)